app/jobs/autotune/build_job.rb

```diff
@@ -1,6 +1,7 @@
 require 'work_dir'
 require 'date'
 require 'logger'
+require 'json'
```

Comment:
I was initially doing json validation during build. removed now that it is no longer needed